recipes/antismash/meta.yaml

```diff
@@ -1,28 +1,30 @@
+{% set name = "antiSMASH" %}
 {% set version = "4.2.0" %}
 {% set sha256 = "cf284c2a18ee8d76762a599b6c6d6cadf5a0adefe2078c84875b22c96dd777b2" %}
 
 package:
-  name: antismash
+  name: {{ name|lower }}
   version: {{ version }}
 
 build:
-  number: 0
+  number: 1
 
 source:
   url: https://dl.secondarymetabolites.org/releases/{{ version }}/antismash-{{ version }}.tar.gz
   sha256: {{ sha256 }}
   patches:
-    - smiles.patch [osx]
+    - smiles.patch  # [osx]
 
 requirements:
   host:
-    - python >=2.7,<3.0a0
+    - python >=2.7,<3
```

Comment:
Just FYI - there is "method to this madness". Basically, it's that `3.0a0 < 3.0`. So if you say `<3.0`, then `3.0a0` or `3.0beta1` would be ok, because they are not yet `3.0`. So for upper bound, using the annoyingly cryptic `3.0a0` is actually correct if it's supposed to say `2 point something`. 

In the case at hand, there won't be a Python 3.0a0, so it's just impeding legibility. But if you see it elsewhere, it may make sense, or could at least be argued for. While we hope not to have any one add versions like `2.0a0`, you know how that can go...

Reply:
Thanks for the explanation Elmar! I was indeed wondering why "3.0a0a" was used intead of "3", and your reason behind it makes total sense.

Reply:
In case there is ever a question:

```python
>>> from conda.exports import VersionOrder
>>> VersionOrder('3.0a') < VersionOrder('3.0') 
True
```

That's the function doing it. So it's what we've got to live by. Its mostly PEP compatible, but there are some idiosyncrasies to it. E.g. a special case is
```python
>>> VersionOrder('3.0post') < VersionOrder('3.0') 
False
```